docs/dev.md

```diff
@@ -367,7 +373,7 @@ when you do not want to commit/push too often.
 #### Android
 
 Anytime a beta is tagged, Travis produces an APK in the Android stage.
-In the logs is displayed an URL for the APK that has been built.
+In the logs is displayed a URL for the APK that has been built.
 This APK can then be tested and then pushed and published in the Play
 Developer Console.
 
```

Comment:
par contre il est question de faire un yarn build préalable, ce n'est pas documenté si ?

Reply:
It defined https://github.com/cozy/cozy-banks/blob/master/docs/dev.md#getting-started


Then you can install the dependencies:

```console
$ yarn
```

Develop:

```console
$ yarn start
```

And build:

```console
$ yarn build
```

Reply:
should we exchange the order of the 2 yarn commands?

Reply:
c'est surtout qu'il faut indiquer je pense, pour le develop, qu'un premier build est nécessaire avant le start

Reply:
done